spec/rails_helper.rb

```diff
@@ -0,0 +1,21 @@
+require 'spec_helper'
+ENV['RAILS_ENV'] ||= 'test'
+require File.expand_path('../config/environment', __dir__)
+
+abort("The Rails environment is running in production mode!") if Rails.env.production?
+require 'spec_helper'
+require 'rspec/rails'
+  
+ActiveRecord::Migration.maintain_test_schema!
+
+  Dir[Rails.root.join("spec/support/**/*.rb")].each { |f| require f }
+
+  RSpec.configure do |config|
+  config.fixture_path = "#{::Rails.root}/spec/fixtures"
+  config.use_transactional_fixtures = true
+  config.infer_spec_type_from_file_location!
+  config.filter_rails_from_backtrace!
+  config.include FactoryBot::Syntax::Methods
+  config.include(Shoulda::Matchers::ActiveRecord, type: :model)
```

Comment:
Please stick to one format. Either leave the parenthesis here and remove above and below or remove them here.

Reply:
Also, please make sure to format your code before you submit it for review.

Reply:
> Please stick to one format. Either leave the parenthesis here and remove above and below or remove them here.

Can you specify what you mean?

Course material has it like this:

# spec/rails_helper.rb
# [...]
RSpec.configure do |config|
  # [...]
  config.include FactoryBot::Syntax::Methods
  config.include(Shoulda::Matchers::ActiveRecord, type: :model)
end

Reply:
```
config.include Shoulda::Matchers::ActiveRecord, type: :model
```